husky_components/tabs/main.js

```diff
@@ -54,7 +55,8 @@ define(function() {
             preSelectEvent: {
                 enabled: false,
                 triggerSelectItem: true
-            }
+            },
+            tabChangeAllowed: null
```

Comment:
Does this component have to know about this at all? Isn't it the job of the component using the tabs to determine if the switch should really happen or not?